packages/terra-clinical-result/translations/en-AU.json

```diff
@@ -17,5 +17,6 @@
   "Terra.clinicalResult.interpretationPositive": "Positive",
   "Terra.clinicalResult.interpretationAbnormal": "Abnormal",
   "Terra.clinicalResult.interpretationHigh": "High",
-  "Terra.clinicalResult.interpretationLow": "Low"
+  "Terra.clinicalResult.interpretationLow": "Low",
+  "Terra.clinicalResult.bloodPressure": "Blood Pressure"
```

Comment:
Is there a translations Jira to update these translations in the future?

Reply:
Yup, the translations would be done as part of TRANS-14531. See this [comment](https://jira.cerner.com/browse/TRANS-14531?focusedCommentId=4752846&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-4752846).

Reply:
Updated the translations for non English languages - https://github.com/cerner/terra-clinical/pull/868/commits/3f28c70e6aa77ceda8d00cb3a50ae80f362ce9b5